### PR TITLE
Creating home & graphs & skeleton

### PR DESCRIPTION
**Pull request by Jasper123pyah**
In this PR I have made the skeleton of the application. After that I have created 2 graphs to show the data. Lastly I have made a home page with hardcoded mockdata.

### CODE REVIEW

**Review by NealGeilen — APPROVED** [review]

File: .idea/vcs.xml [hunk 1]
```diff
@@ -2,5 +2,6 @@
 <project version="4">
```
Comment (Typiqally) [inline .idea/vcs.xml:2]: Remove this file from the repo, add to .gitignore

File: interfaces/data.ts [hunk 2]
```diff
@@ -0,0 +1,14 @@
+export interface dataSeries {
```
Comment (Typiqally) [inline interfaces/data.ts:1]: Use pascal case

File: pages/_app.tsx [hunk 3]
```diff
@@ -1,8 +1,14 @@
 import '../styles/globals.css'
-import type { AppProps } from 'next/app'
+import type {AppProps} from 'next/app'
+import Header from "../components/shared/Header";
 
-function MyApp({ Component, pageProps }: AppProps) {
-  return <Component {...pageProps} />
+function MyApp({Component, pageProps}: AppProps) {
```
Comment (Typiqally) [inline pages/_app.tsx:5]: Rename this to Application, MyApp looks a bit like boilerplate

File: pages/index.tsx [hunk 4]
```diff
@@ -1,24 +1,28 @@
...
-      <div className={styles.container}>
-          <Head>
-              <title>QS Dashboard</title>
-              <meta name="description" content="Quantified Student Dashboard"/>
-              <link rel="icon" href="/favicon.ico"/>
-          </Head>
+    return (
+        <div className={styles.container}>
+            <Head>
+                <title>QS Dashboard</title>
```
Comment (Typiqally) [inline pages/index.tsx:10]: Is it necessary to put <Head>on every page? Can't we create a default and override if necessary.

**Review by Typiqally — CHANGES_REQUESTED** [review]

**Review by Typiqally — APPROVED** [review]
LGTM

**Review by NealGeilen — APPROVED** [review]

**Pull request merged**